src/olympia/migrations/917-drop-old-addons-columns.sql

```diff
@@ -0,0 +1,30 @@
+DROP PROCEDURE IF EXISTS drop_addons_col_if_exists;
+
+DELIMITER ';;'
+
+CREATE PROCEDURE drop_addons_col_if_exists() BEGIN
+    /* drop the known latest version foreign keys if they exist */
+    IF EXISTS (SELECT * FROM information_schema.table_constraints WHERE table_schema = (SELECT DATABASE()) AND table_name = 'addons' AND constraint_name = 'latest_version_refs_versions') THEN
+        ALTER TABLE addons drop foreign key latest_version_refs_versions;
+    END IF;
+    IF EXISTS (SELECT * FROM information_schema.table_constraints WHERE table_schema = (SELECT DATABASE()) AND table_name = 'addons' AND constraint_name = 'latest_version_refs_id_46aa95ab') THEN
+        ALTER TABLE addons drop foreign key latest_version_refs_id_46aa95ab;
```

Comment:
It's what django generates, so it should be consistent across all envs that used syncdb (you can check locally if you still have it); the one above, `latest_version_refs_versions` is what we had on dev/stage/prod for some reason.